pkg/itf/README.md

```
@@ -274,8 +268,8 @@ suite.GET("/test").
     ExpectOK()
 ```
 
-### Backwards Compatibility
-All existing ITF code continues to work without modification.
+### Compatibility Policy
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)